### ISSUE TITLE
Reference database creation

### ISSUE THREAD

**Issue by jensfriisnielsen** [gyper/gyper#7]
First of all, what a great tool!
I have been unable to find any contact information @hannespetur, but would like to request a feature. Right now there are only a limited number of alleles available as reference. Would it be possible to create more pre-formatted alleles, or provide the source code that you used to create them? In that way if there was a way to update the references, gyper could easily rival the industry standards in HLA-typing. You could put the scripts/code in a develop branch, or the like, if they need updates to be functional. In that way it would be possible for other users to create and maintain larger databases and that would be tremendously helpful.
Please don't hesitate to contact me at gmail.com, same name
Again thank you for a great tool!

**Comment by hannespetur** [comment]
Hi Jens

I am happy to know you have found gyper useful.

To create the HLA references I wrote a XML wrapper using the RapidXML library for the IMGT/HLA database. The wrapper creates uncapped FASTA files but using MUSCLE I then multi sequence aligned the sequences to add the gaps. IIRC we decided not to include the code for this in this repository because it relied on external programs and libraries. However, if you think it would help you I will gladly look these codes up and add them here.

Btw, I have changed my display settings so that my email address is now visible to others on Github.

Hannes
